packages/flame/lib/src/camera/viewfinder.dart

```diff
@@ -19,21 +18,17 @@ import 'package:meta/meta.dart';
 /// If you add children to the [Viewfinder] they will appear like HUDs i.e.
 /// statically in front of the world.
 class Viewfinder extends Component
```

Comment:
why no longer parent is a camera?

Reply:
Initially it wasn't there, then I added it in the last PR, but some operations need to check whether it has a parent or not and `parentIsA` does a `!` on the parent, so we couldn't do those checks anymore which led to some bugs.